beacon-chain/state/testtmpl/getters_validator.go

```diff
@@ -0,0 +1,22 @@
+package testtmpl
```

Comment:
```suggestion
package testing
```
or `test`

Reply:
@terencechain my idea is to create a package not general for testing, but for the specific goal of having all the test code that are repeated in many subpackages of state

Reply:
testing here refers to the test template too; it’s more idiomatic and follows the rest of the repo